flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/observer/Observer.java

```diff
@@ -46,6 +46,8 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(Observer.class);
 
+    public static final String JOB_STATE_UNKNOWN = "UNKNOWN";
```

Comment:
Does this need to add the enum `UNKOWN` in `JobState` and directly use this enum?

Reply:
No, this has nothing to do with expected job states. This means that we don't know the current state. We expected state (JobState enum) is always known.